tools/extras/install_tensorflow_cc.sh

```diff
@@ -35,15 +36,16 @@ cd ../
 # now bazel is built
 git clone https://github.com/tensorflow/tensorflow
 cd tensorflow
+git reset --hard 49059695babc08df22f31124480142e7f6aec5eb
```

Comment:
I highly recommend `git checkout 49059695babc08df22f31124480142e7f6aec5eb` instead. It's non-destructive and more intuitive to seasoned git users.

I notice this version is particularly cutting edge. Any particular reason for this as opposed to a stable version like v1.4.0?